Cargo.lock

```diff
@@ -3175,7 +3175,7 @@ dependencies = [
  "rustc-hash",
  "rustc-rayon 0.2.0",
  "rustc-rayon-core 0.2.0",
- "smallvec",
+ "smallvec 0.6.10",
```

Comment:
I suppose this will end up automatically upgrading itself when this crate is auto-published?

Reply:
Yes, but bumping the rustc-ap crates in rls/rustfmt still needs to happen manually.